meetings/2025-04-23.md

```diff
@@ -0,0 +1,111 @@
+
+---
+
+## Topics
+
+### Solid Symposium 2025
+
+https://sosy2025.eu/
+
+* eP: April 24-25
+* eP: Will there be materials available online afterwards?
+* ... I believe Jeff, Rahul and Jesse have sessions.
+* RG: Solid Practitioners are organizing a session there. There are 6 talks. We try to foster collaboration between communities.
+* ... Speakers from EU and US. 
+* https://sosy2025.pages.dev/solid-together
+* eP: Good luck.
+* RG: It's tomorrow morning 10am CET.
+* ... I intend to put up the slides. We probably won't have video. We'll try to broadcast, but we won't have the ability to share the link too much in advance.
+
+### Code of Conduct related issues
+
+* eP: Tomorrow Hadrian and I will have a call with W3C staff. Michiel is on vacations.
+* eP: If you would like to bring up interactions where CoC might have been violated. You can do it publically or contact CG chairs directly. 
+* RG: Please refrain from discussing any issue that involves me until I am sent an official communication from W3C and the Solid CG.
+* HZ: Based on my experience with Apache, we talk about issues not people in public. In general people mean well just emotions get escalated. I think we will resolve it and my goal is to prevent such problems from haunting us in the fugure. We are a community and we have to communicate.
+* HZ: In general I don't want to offer to mediate, unless I'm asked. This might be seen as pressure from one of the sides. Please ask me if you want help with resolving the problem.
+* RG: I haven't received any communication except the two public comments. I don't think the issues should be discussed until the situation is resolved. I have right to privacy and ask for a right to privacy. 
+* eP: We are trying to update everybody, in the minutes and on matrix. It's not about a person in particular, it's about the whole situation.
+* HZ: I don't think we can resolve anything now. Rahul your point was understood.
+* eP: We'll know better how to move forward after tomorrow's meeting. I think it's important to have a good working environment.
+
+### Backup/Restore Issue history
+
+https://github.com/solid/specification/issues?q=is%3Aissue%20state%3Aopen%20%22copyright%22%20in%3Atitle%20
+
+https://github.com/solid/specification/issues/454#issuecomment-2666339495
+
++ RG: Some previous participants have been deleting their comments in a high-handed manner (I am not against somene making an edit or even deleting a comment because they changed their minds). But when it is done after many years, this behaviour is disruptive. Of specific concern is the deletion headings in theses issues. 
++ RG: Valuable loss of context for those who joined the CG later. Can we please restore those comments, unless there is some exceptional legal reason to let them remain deleted. 
++ RG: The person is not even a CG member when they deleted the issues.
+* ... One person removed comments that were written years ago.
```

Comment:
```suggestion
+ RG: Some previous participants have been deleting their comments in a high-handed manner. (I am not against someone making an edit or even deleting a comment because they changed their minds.) But when it is done after many years, this behaviour is disruptive. Of specific concern, is the deletion headings in these issues. 
+ ... Valuable loss of context for those who joined the CG later. Can we please restore those comments? Unless there is some exceptional legal reason to let them remain deleted. 
+ ... The person is not even a CG member when they deleted the issues.
* ... One person removed comments that were written years ago.
```